README.md

```diff
@@ -14,7 +14,7 @@ Okta's Spring Boot Starter will enable your Spring Boot application to work with
 
 * An Okta account (sign up for a [forever-free developer account](https://developer.okta.com/signup/))
 * An OIDC application (typically a 'Web' application)
-* An [access token](https://developer.okta.com/docs/api/resources/oauth2.html)
+* An [access token](https://developer.okta.com/docs/guides/implement-implicit/use-flow/)
```

Comment:
We shouldn't link to an implicit flow doc, we should find a resource server doc, or a specific page that talks about getting a token

Reply:
Linked to `https://developer.okta.com/docs/guides/implement-oauth-for-okta/request-access-token/`.